package.json

```diff
@@ -29,12 +29,11 @@
     "algoliasearch": "^4.2.0",
     "balance-text": "^3.3.0",
     "bl": "4.0.3",
-    "bootstrap": "4.5",
-    "gatsby": "^2.25.3",
-    "gatsby-background-image": "^0.9.19",
+    "bootstrap": "4.6.0",
+    "gatsby": "^2.31.1",
     "gatsby-cli": "^2.12.93",
     "gatsby-image": "^2.4.1",
-    "gatsby-plugin-algolia": "^0.13.0",
+    "gatsby-plugin-algolia": "^0.16.2",
```

Comment:
Seeing this reminded me - this actually needs to be hard-coded to "0.13.1". Releases higher than that have a bug which wipes out your index settings due to a change they made. I started working with them to fix the bug, but the fix hasn't made it in yet

Reply:
@epbarger According to https://www.npmjs.com/package/gatsby-plugin-algolia there's actually no v0.13.1, so I'm going to peg it to 0.13.0, is that OK?